.gitignore

```diff
@@ -29,3 +29,4 @@
 /node_modules
 /yarn-error.log
 .cache/
+.DS_Store/
```

Comment:
This would need to be more generic to catch all instances of this file as it occurs in many directories and subdirectories (e.g. the one in the `app` directory has survived). I'm not sure we should need to add this though if we make sure we don't commit those files.